src/main/java/odd/jobs/services/validator/components/PasswordValidator.java

```diff
@@ -0,0 +1,17 @@
+package odd.jobs.services.validator.components;
+
+import odd.jobs.entities.user.User;
+
+public class PasswordValidator implements IValidator {
+
+    @Override
+    public String validate(User user) {
+        String attribute = user.getPassword();
+        if(attribute.length() > 30){
+            return "password is too long";
+        } else if(attribute.length() < 3) {
+            return "password is too short";
+        }//TODO check for whitespace
```

Comment:
biale spacje sa chyba normalnymi znakazmi, chyba ze chodzi Ci zeby haslo nie zawierało tylko spacji?

Reply:
spoko byłoby od razu to zrobić, zeby juz do tego nie wracac